src/Game/ETG.c

```diff
@@ -6166,21 +6095,16 @@ sdword etgFunctionCall(Effect *effect, struct etgeffectstatic *stat, ubyte *opco
 
     if ((memsize)((etgfunctioncall *)opcode)->function == (memsize) etgSpawnNewEffect){
 
-//        dbgMessagef("Function: %lx:%d   -> %s",((etgfunctioncall *)opcode)->function, nParams+((etgfunctioncall *)opcode)->passThis ,  stat->name);
-
         switch  (nParams+((etgfunctioncall *)opcode)->passThis){
```

Comment:
Maybe it's not relevant to this MR, but this seems like it could be refactored like you did with the other switch statement in `wrapped_unlisted_functions.h`, right?

Reply:
Yes well, as can see on line 5983, this is the _x86_64 specific code, and indeed the switch cases do look alike. I suspect the generic code was inspired by this one.

With `--enable-generic-etg` this code isn't even compiled, so I didn't bother. Actually, since the generic code works better than this assembly code now, I was even considering deprecating this.

As for the commented code that I deleted, I needed to prune it so I could compare the two versions side by side without useless clutter (the generic one and the x86_64 one).